README.md

```diff
@@ -114,10 +149,68 @@ In case of problems with the build, please tell us including the following thing
 - CMakeCache.txt
 - If you are using windows, please include the output of ```cygcheck -s ```
 
-## Source Directory Structure
+
+# Basic Usage
+
+Once compiled, the VM entry point is at `./build/dist/pharo`.
+Since PharoVM is only a VM, an image is required to do useful things.
+
+An image used for the build process is available at `./build/vmmaker/image/Pharo*.image` (glob here because the name is variable since it contains version number and hash).
+
+```bash
+$ ./build/dist/pharo ./build/vmmaker/image/Pharo*.image eval '3+4'
+7
+```
+
+Or run the graphical environment with.
+
+```bash
+$ ./build/dist/pharo ./build/vmmaker/image/Pharo*.image --interactive
+```
+
+Note: since this image is used by the build process, it is not advised to alter it, but you can still save it as a copy.
+
+## Hacking
+
+### Editing the VM code in your image
+
+You can load the Pharo code of the VM using Pharo's git client Iceberg.
+You can do so by cloning directly this repository from Iceberg, or by adding an already existing clone to it.
+
+Alternatively, if you're building the VM using the instructions above, the build process does already generate a Pharo image with the VM code loaded. You'll find such image at `./build/vmmaker/image/VMMaker.image` inside your build directory.
+
+### Compiling a new VM
+
+The build process is described above, so in order to generate a new PharoVM with your changes, there is different approaches.
+
+* Commit your changes (with Iceberg) into the local git repository you cloned then run `make` from the building directory.
+  This will rebuild a new VMMaker image by loading your new code into the basic Pharo image.
+  Note: this rebuild starts at the step 3.
+
+* Save your image as XXX (or overwrite the `VMMaker.image` file), then run:
+
+  `cmake ????` TODO
```

Comment:
I'm not sure this works right now.
The Cmake process will recreate and overwrite the VMMaker.image file :/

Reply:
Oh. I assumed there were some flags to skip the VMMaker creation.

Reply:
OK. I have a PR to enable this #482